janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphTest.java

```diff
@@ -5077,6 +5081,8 @@ public void read(Message message) {
                         //v2 deletion transaction
                         assertEquals(0, Iterables.size(changes.getVertices(Change.ADDED)));
                         assertEquals(1, Iterables.size(changes.getVertices(Change.REMOVED)));
+                        String vertexLabel = changes.getVertices(Change.REMOVED).iterator().next().label();
+                        assertEquals(testVertexLabel, vertexLabel);
```

Comment:
This two lines would fail without the business logic change because `vertexLabel` here would be equal to `vertex` even so real label of the vertex label is `testVertex`. With this PR this vertex now returns correct vertex label.